conduct.md

```diff
@@ -116,15 +119,23 @@ how long a decision will take to implement.
     * Report errors or bugs to the appropriate team members.
     * When a choice is disputed, use Github issue tracker and let team members
       share their opinion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for the decision making as a team.
+    * Defer to the Professor or Technical Leaders when unsure about tasks or implementation.
+    * Use efficient communication within and outside of teams.
+    * Communicate with the appropriate team member about any changes that is needed to be done to any project.
+    * Use efficient communication within and outside of teams
+    * Stay respectful and openminded when discussing decisions and ideas.
+    * Communicate with the appropriate team member about any changes that is needed to be done to any project.
 
 * Unacceptable Conduct (U):
   * Members shall not
     * Let an individual team member or a project manager make all major decisions.
     * Make decisions without finding evidence showing that it would likely be beneficial.
     * Refuse to work on the decided option.
     * Make decisions that would affect another team without first notifying them.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for the decision making as a team.
+    * Refuse to actively participate in group discussions or breakout rooms.
+    * Deploy any kind of decision that's against codes of ethic
+    * Mock a team member's decision
+    * Refuse to fix any bugs or technical issues in the project.
```

Comment:
Suggestion: consider rephrasing this sentence to include "assigned work in general" instead of just bugs and technical issues. This is not a request and if you don't agree with this suggestion then that's completely fine.